english/normal-dungeon/4/4.md

```diff
@@ -1,5 +1,5 @@
-You open the door to your neighbour's house, and the first thing you see is a trail of blood💉 streaked on the floor...
+ You open the door 🚪 to your neighbour's house 🏡, and the first thing you see is a <span style= "color:red">trail of blood💉 streaked on the floor...</span>
```

Comment:
DO NOT use HTML code on the markdown file. if you are more advanced and know ReactJs you could beautify the page